app/templates/discover.hbs

```diff
@@ -106,8 +106,9 @@
                 </div>
 
                 <div class="m-v-md registration-type-selector">
-                    <i class='type-selector-warning' style='color: red; display: none'>Only available with OSF Registries</i>
-                    <h4 class="filter-heading p-sm">{{t "discover.main.type"}}</h4>
+                    <h4 class="filter-heading p-sm" aria-label={{t "discover.main.osf_registraton_filter"}}>
+                        {{t "discover.main.type"}} <span class="hint--bottom hint--small" data-hint={{t "discover.main.popover.registries.body"}}><i class="fa fa-question-circle permission-info" aria-label={{t "discover.main.popover.registries.body"}}></i></span>
```

Comment:
At 120% zoom, this is oddly truncated.

Consider using `hint--right` and `hint--large` instead, to give the tooltip more room to show the long sentence text.
<img width="149" alt="screen shot 2017-03-14 at 12 00 07 pm" src="https://cloud.githubusercontent.com/assets/2957073/23910288/8913c4c6-08af-11e7-95e7-14f0a44cacbc.png">